.github/workflows/semantic.yml

```diff
@@ -7,8 +7,7 @@ on:
       - edited
       - synchronize
 
-permissions:
-  contents: read
```

Comment:
Do we need to push this down into the job permissions, or do we think it's unnecessary for this action?